cmd/zfs/zfs_main.c

```diff
@@ -897,7 +911,7 @@ zfs_do_create(int argc, char **argv)
 		nomem();
 
 	/* check options */
-	while ((c = getopt(argc, argv, ":V:b:so:p")) != -1) {
+	while ((c = getopt(argc, argv, "PV:b:nso:pv")) != -1) {
```

Comment:
nit: the leading `:` should be kept here, it's for the `':'` case below when an optional argument is missing.  From `getopt(3)`.

```
       If  the  first  character  (following any optional '+' or '-' described
       above) of optstring is a colon (':'), then getopt() returns ':' instead
       of  '?'  to  indicate  a  missing  option  argument.
```

Reply:
oops, will fix.